src/axom/slic/interface/slic_macros.hpp

```diff
@@ -72,6 +72,49 @@
     }                                                               \
   } while(axom::slic::detail::false_value)
 
+/*!
+ * @brief Macro that logs given error message only on root.
+ * \note Must call `axom::slic::initialize(is_root=true)`
+ *   or set via `axom::slic::setIsRoot(true)`. Otherwise, this not filter based on root.
```

Comment:
Clarification (here and elsewhere)

Please update to something like the following (but better):
```suggestion
 * \note By default, all ranks are considered to be root.
 *       Must call `axom::slic::initialize(is_root={true|false})`
 *       or set via `axom::slic::setIsRoot({true|false})` to filter based on root.
```